upload-riffraff.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Got it, I haven't used build matrices and hadn't thought through the consequences. That makes sense.